scripts/convert_pixart_alpha_to_diffusers.py

```diff
@@ -180,14 +180,14 @@ def main(args):
 if __name__ == "__main__":
     parser = argparse.ArgumentParser()
 
-    parser.add_argument(
-        "--orig_ckpt_path", default=None, type=str, required=False, help="Path to the checkpoint to convert."
-    )
+    # set multi_scale_train=True if using PixArtMS structure during training else set it to False
+    parser.add_argument("--multi_scale_train", default=True, type=str, required=True, help="If use Multi-Scale PixArtMS structure during training.")
```

Comment:
The type is `str` and we're defaulting to a `bool`. This needs to be fixed.

Reply:
Also, do we have any other 1024x1024 checkpoints that are affected by this? If not, do we really need this flag?